docs/client-and-adapters.md

```diff
@@ -108,6 +108,8 @@ htmlFooter();
 
 ```
 
+See [#397](https://github.com/solariumphp/solarium/issues/397#issuecomment-727662811) on how to leverage the [Symphony PSR-18 HTTP Client](https://symfony.com/doc/current/http_client.html#psr-18-and-psr-17) to reuse Solr connections.
```

Comment:
you might want to give the example from your comment here rather than referencing it?

Reply:
@wickedOne Sorry for the silence. I went ahead with a proposal to extend the documentation for the PSR-18 Adapter, addressing:

- Custom timeouts
- Connection reuse
- Code additional example
- Note on open file handles "leak"

Feedback/suggestions welcome.